tv-shows-app/src/components/feature/review/ReviewItem/components/ReviewUpdate/ReviewUpdate.tsx

```diff
@@ -0,0 +1,47 @@
+import { CloseIcon, EditIcon } from "@chakra-ui/icons";
+import { IconButton, Modal, ModalBody, useDisclosure, Text, ModalContent, ModalOverlay, ModalHeader, CloseButton, ModalCloseButton } from "@chakra-ui/react";
+import ReviewForm from "../../../ReviewForm/ReviewForm";
+import { IReview } from "@/typings/review";
+import useSWRMutation from "swr/mutation";
+import { swrKeys } from "@/fetchers/swrKeys";
+import { updateReview } from "@/fetchers/mutators";
+import { mutate } from "swr";
+
+export interface IReviewUpdateProps {
+   updatingReview: IReview
+} 
+
+export default function ReviewUpdate({updatingReview} : IReviewUpdateProps) {
+   const {isOpen, onOpen, onClose} = useDisclosure();
+   const onOpening = () => {
+      console.log("clicked");
+      onOpen();
+   }
+
+   const {trigger} = useSWRMutation(swrKeys.reviews(`/${updatingReview.id}`), updateReview, {
+      onSuccess: () => {
+         mutate(swrKeys.getReviews(updatingReview.show_id))

```

Comment:
Opet, bonus bonusa ovdje:
Ako se sjecas sto smo pricali, ajmo mu probati mi reci koji je novi cache za prvu ruku (da nam bude malo brzi render).
Ako nisi sigurna kako tocno, javi se prodjemo skupa :)

Reply:
Isto to mozes napraviti i za delete i create